### ISSUE TITLE
Option to change field display (dashed border-bottom)

### ISSUE THREAD

**Issue by jansportw** [vitalets/x-editable#403]
Would be nice if could change/define (from within editable(options) or data-) the way a editable field is displayed prior to being selected. Yes, CSS is the best way to make fine tuned changes, but it would be nice to make less fine tuned changes field by field. Such as simply choosing between dashed or nothing, or even just entering a color code that will override the default dash color 

On a page with lots of editable fields, that many dashes is overwhelming.

**Comment by Hexodus** [comment]
I agree with jansportw - I'm in exact this situation. I need to disable this underline only for a specific textarea because this effect is to massive and distracting.

**Comment by bhoriuchi** [comment]
@Hexodus you can just use a little jquery
<pre>
&lt;span id="myeditable" data-underline="false"&gt;edit me!&lt;/span&gt;
</pre>

```
$('#myeditable').editable();
$('[data-underline="false"]').css('border-bottom', 'none');
```

**Comment by artforlife** [comment]
Has this, perchance, been implemented? It seems like a really useful customization.

**Comment by blsiha** [comment]
#myEditableTextArea.editable-click {
    border-bottom-color: #cccccc;
    border-bottom-style: solid;
}
